CONTRIBUTING.md

```diff
@@ -8,7 +8,7 @@ agree to abide by the [Contributor Covenant](http://contributor-covenant.org).
 1. [Fork](https://help.github.com/articles/fork-a-repo/) this project
 2. [Clone](https://help.github.com/articles/fork-a-repo/#step-2-create-a-local-clone-of-your-fork) your forked version `git clone git@github.com:<YOUR-USERNAME>/hacktoberfest-practice.git`
 3. Make changes
-4. [Commit](https://help.github.com/articles/adding-a-file-to-a-repository-using-the-command-line/) your changes (write a short descriptive message of what you have done)
+4. Agree to abide by the [Contributor Covenant](https://www.contributor-covenant.org/).
```

Comment:
```suggestion
4. [Commit](https://help.github.com/articles/adding-a-file-to-a-repository-using-the-command-line/) your changes (write a short descriptive message of what you have done)
```

Reply:
wait, this is literally the older version I guess